packages/playground/_stories/main/Avatar/Avatar.stories.ts

```diff
@@ -36,6 +36,7 @@ const Template: UI5StoryArgs<Avatar, StoryArgsSlots> = (args) =>
     color-scheme="${ifDefined(args.colorScheme)}"
     ?interactive="${ifDefined(args.interactive)}"
     aria-haspopup="${ifDefined(args.ariaHaspopup)}"
+    accessible-name="Avatar with accessible name"
```

Comment:
This could be made dynamic by outside args object as it would be better to have it only if interactive is true.

```html
accessible-name="${ifDefined(args.accessibleName)}"
```

```js
Basic.args = {
  initials: "FJ",
  interactive: true,
  accessibleName: "Avatar with accessible name"
};